bin/start_backend.sh

```diff
@@ -206,14 +215,47 @@ fi
 if [ ${RUN_LOG_CONSOLE} -eq 1 ] ; then
     # force glog output to console (stderr)
     export GLOG_logtostderr=1
-else
```

Comment:
don't remove this, it'll break the assumption that crash core will be redirected to be.out/cn.out